ru/turing/courses/lesson2/animals/Animal.java

```diff
@@ -1,4 +1,59 @@
 package ru.turing.courses.lesson2.animals;
 
-public class Animal {
-}
+public abstract class Animal {
+
+    protected String vid;
```

Comment:
```suggestion
    protected String type;
```
Называем английскими словами везде, где возможно